xclim/indicators/atmos/_temperature.py

```diff
@@ -1518,4 +1518,5 @@ def cfcheck(self, tas, snd=None):
     long_name="Chill units after the Utah Model",
     allowed_periods=["Y"],
     compute=indices.chill_units,
+    parameters=dict(positive_only={"default": False, "kind": InputKind.BOOL}),
```

Comment:
```suggestion
```
I think this line is not needed if the proper annotation is added in the function.

Reply:
Makes total sense awesome 👍